### ISSUE TITLE
Not uploaded to haxelib?

### ISSUE THREAD

**Issue by fourst4r** [Aurel300/ammer#54]
I noticed this library is not uploaded to lib.haxe.org, is that intentional?

**Comment by Aurel300** [comment]
Yes, for now. I will probably put `ammer` and `ammer-core` on haxelib once #53 is completed.
